### PR TITLE
IP-1 Create new Database models and refactor code

### PR DESCRIPTION
**Pull request by gdatudore**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
-db = SQLAlchemy(app)
-
-class Hub(db.Model):
-    id = db.Column(db.Integer, primary_key = True)
-    name = db.Column(db.String(80), unique=True, nullable=False)
-    user_number = db.Column(db.Integer)
-
-    def __repr__(self):
-        return f'{self.id}, {self.name}, {self.user_number}'
+from model import *
```
Comment (alexanderdx) [inline app.py:2]: can we get rid of the wildcard import?

File: app.py [hunk 2]
```diff
@@ -36,7 +22,7 @@ def get_hub_by_id(id):
 
 @app.route('/hubs', methods=['POST'])
 def add_hub():
-    hub = Hub(id=request.json['id'], name=request.json['name'], user_number=request.json['user_number'])
```
Comment (alexanderdx) [inline app.py:39]: is the db field auto-incrementing for this to work?

**Review by alexanderdx — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (gdatudore) [reply app.py:2]: No, we can't :).

File: app.py (hunk 2 above)
Comment (gdatudore) [reply app.py:39]: Yes

**Pull request merged**